src/deps.txt

```diff
@@ -1,3 +1,6 @@
+# These packages are what coreos-assembler needs; they will
+# be installed in the container image.
```

Comment:
Now that #1018 merged, probably also worth something like: "Note that packages in `vmdeps.txt` are also installed." ?